server/index.js

```diff
@@ -62,7 +62,7 @@ io.on('connection', (socket) => {
         console.log('getting top 10')
         getTop10().then((top10) => {
             // TODO add call to redis to initialize swiping room here
-            io.to(user.room).emit('sessionMembers', { room: user.room, users: getUsersInRoom(user.room), host: getHost(user.room), top10: top10});
+            io.to(user.room).emit('sessionMembers', { roomId: user.room, users: getUsersInRoom(user.room), host: getHost(user.room), top10: top10});
         }).catch((error) => {
             console.log('error getting list of top 10 movies from rapidapi')
             callback(error);
```

Comment:
Is callback defined in this method? Would it return an error if it isn't defined? Let me know your thoughts.

Reply:
callback is already defined in the outer socket call `socket.on('begin', ( user , callback) => { ...`. I believe there wouldn't be an issue taking it out since its called in the outer socket function (line 70). Perhaps the implementer would have an opinion about this though. @dlhxu what are your thoughts?

Reply:
@billxsheng We do currently have the callback defined in the socket emit in [Lobby.js](https://github.com/rosakng/wewatch/blob/a2587064f4a74aebc2eaa3b2fec9d0927d81c446/wewatch-client/src/components/Lobby/Lobby.js#L19). I believe this should be sufficient to ensure no errors will occur, what are your thoughts?

Reply:
You're right. Thanks for pointing it out. Consider this issue resolved.